services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/UpdateItemEnhancedResponse.java

```diff
@@ -86,19 +97,22 @@ public boolean equals(Object o) {
         UpdateItemEnhancedResponse<?> that = (UpdateItemEnhancedResponse<?>) o;
         return Objects.equals(attributes, that.attributes)
                && Objects.equals(consumedCapacity, that.consumedCapacity)
-               && Objects.equals(itemCollectionMetrics, that.itemCollectionMetrics);
+               && Objects.equals(itemCollectionMetrics, that.itemCollectionMetrics)
+               && Objects.equals(responseMetadata, that.responseMetadata);
     }
 
     @Override
     public int hashCode() {
         int result = Objects.hashCode(attributes);
         result = 31 * result + Objects.hashCode(consumedCapacity);
         result = 31 * result + Objects.hashCode(itemCollectionMetrics);
+        result = 31 * result + Objects.hashCode(responseMetadata);
         return result;
     }
 
     @NotThreadSafe
     public static final class Builder<T> {
+        public DynamoDbResponseMetadata responseMetadata;
```

Comment:
Should be `private` as in DeleteItemEnhancedResponse 

Nit: put it as the last instance variable for consistency